internal/grpcwrap/provider.go

```diff
@@ -121,6 +121,9 @@ func (p *provider) ValidateResourceTypeConfig(_ context.Context, req *tfplugin5.
 	validateResp := p.provider.ValidateResourceConfig(providers.ValidateResourceConfigRequest{
 		TypeName: req.TypeName,
 		Config:   configVal,
+		ClientCapabilities: providers.ClientCapabilities{
+			WriteOnlyAttributesAllowed: true,
```

Comment:
This may not be related entirely to your PR but do we not need to be passing the other capability here as well? i.e. `DeferralAllowed`?

Reply:
We did not before so false must have been the right value for those fields.

Reply:
I'm guessing it only didn't matter because deferral wasn't important in the context of validation but I'd argue it's a bug that we didn't pass it before?

Reply:
I don't think a validation can be deferred, so deferrals allowed should be false, which it is by default. Let's talk about this async after the holidays in more detail, I'm going to merge this so that the change can be in the alpha release 👍